tests/ui/pages/management/tags-page.ts

```diff
@@ -22,6 +23,14 @@ export default class TagsPage extends ManagementPage {
     return new List(this.page, SELECTORS.tagLineRoot, SELECTORS.tagLineItem);
   }
 
+  public normalTag(tagName: string): Locator {
+    return this.page.locator(SELECTORS.tagString(tagName));
+  }
+
+  public importantTag(tagName: string): Locator {
+    return this.normalTag(tagName).locator(SELECTORS.tagImportantMark, { hasText: 'important' });
```

Comment:
you can use PW text selectors like :has-text
https://playwright.dev/docs/selectors#text-selector